lib/stripe/api_resource.rb

```diff
@@ -23,7 +23,9 @@ def refresh
       refresh_from(response, api_key)
     end
 
-    def self.retrieve(id, api_key=nil)
+    def self.retrieve(*args)
```

Comment:
I'm a bit bearish on having function signatures like this, as it makes it harder to tell at a glance what the arguments should be.

There's a nice alternate approach though that I'd be in favor of:

```
ARGUMENT_NOT_PROVIDED = Object.new
def self.retrieve(id, api_key=ARGUMENT_NOT_PROVIDED)
  if api_key.equal?(ARGUMENT_NOT_PROVIDED)
    api_key = nil
  else
    Util.check_bad_api_key!(api_key)
  end
end
```